daemon/cmd/daemon_main.go

```diff
@@ -542,6 +542,9 @@ func init() {
 	flags.Bool(option.EnableHostFirewall, false, "Enable host network policies")
 	option.BindEnv(option.EnableHostFirewall)
 
+	flags.String(option.IPv4NativeRoutingCIDR, "", "Allows to explicitly specify the CIDR for native routing. This value corresponds to the configured cluster-cidr.")
```

Comment:
Not related to your PR, but why we didn't call the flag `--cluster-cidr`?

Reply:
Because it is not the cluster CIDR. The native routing space might be much larger, spanning multiple clusters or even non-k8s space.